scripts/setup_orchestrator_postgres_db.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+. "$(dirname "$0")"/_select_docker_compose.sh
+. "$(dirname "$0")"/_load_dot_env.sh $1
+
+# Deploy postgres omotes schema
+$DOCKER_COMPOSE -f ${2} --profile=manual_dev down orchestrator_postgres_db_dev
+$DOCKER_COMPOSE -f ${2} up -d --wait orchestrator_postgres_db
+
+$DOCKER_COMPOSE -f ${2} exec orchestrator_postgres_db psql -d postgres -c 'CREATE DATABASE omotes_jobs;'
```

Comment:
Probably remove `-d` here since you need the database present in the next commands.

Reply:
We are referencing the default database as psql requires a database to connect to.